thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/ThirdEyeDashboardConfiguration.java

```diff
@@ -11,7 +11,9 @@
   String rcaConfigPath;
   int rcaParallelism;
   List<String> rcaFormatters;
-  String rcaDefaultFramework;
```

Comment:
Don't these need default values? Can dashboard never be started without rca framework?